### PR TITLE
Add example dataset for Baggen 2021

### PR DESCRIPTION
**Pull request by vaofford**

### CODE REVIEW

File: examples/baggen_2021/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+>https://doi.org/10.1038/s41588-021-00805-2   
+
+## Analysing the example dataset with C-SAR
+
+### Clone the C-SAR GitHub repository
+
+To run [C-SAR](https://github.com/cancerit/C-SAR) on the example dataset, you will first need to clone the C-SAR repository:
+
+```
+git clone https://github.com/cancerit/C-SAR.git
```
Comment (keiranmraine) [inline examples/baggen_2021/README.md:16]: This won't work if you need the submodules, also you probably want them to get `main` (git has changed the default name):

```
git clone --recurse-submodules --branch main https://github.com/cancerit/C-SAR.git
```

File: examples/baggen_2021/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+```
+id	sgrna_ids	sgrna_seqs	gene_pair_id
+1	A1BG_1	CATCTTCTTTCACCTGAACG	A1BG
+2	A1BG_2	CTCCGGGGAGAACTCCGGCG	A1BG
+3	A1BG_3	TCTCCATGGTGCATCAGCAC	A1BG
+4	A1BG_4	TGGAAGTCCACTCCACTCAG	A1BG
+5	A2M_1	ACTGCATCTGTGCAAACGGG	A2M
+```
+
```
Comment (keiranmraine) [inline examples/baggen_2021/README.md:94]: Confused, I think the text in the `C-SAR` block should be different.

File: examples/baggen_2021/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+This will write the reformatted FASTQ files to `reformatted_fastq`. 
+
+#### Quantifying samples with pyCROQUET
+
+Below is an example pyCROQUET command for SRR13255539:
+
+```
+pycroquet single-guide -g ${ANALYSIS_DIR}/Brunello_library_Baggen_SupplTable3.reformatted.tsv -q ${ANALYSIS_DIR}/reformatted_fastq/SRR13255539.fastq.gz -s SRR13255539 -o ${ANALYSIS_DIR}/counts/SRR13255539 -n -c 32"
```
Comment (keiranmraine) [inline examples/baggen_2021/README.md:118]: 32 cpus is a bit extreme and likely to cause issues if someone runs it blindly (results are stable regardless of CPU count).  Just add a note to set `-c` appropriately,

**Review by keiranmraine — CHANGES_REQUESTED** [review]
See the comments on the readme file.

File: examples/baggen_2021/README.md (hunk 3 above)
Comment (vaofford) [reply examples/baggen_2021/README.md]: Updated in 05d43cf

File: examples/baggen_2021/README.md (hunk 1 above)
Comment (vaofford) [reply examples/baggen_2021/README.md]: Fixed in 05d43cf

File: examples/baggen_2021/README.md [hunk 4]
```diff
@@ -92,6 +92,8 @@ id	sgrna_ids	sgrna_seqs	gene_pair_id
 5	A2M_1	ACTGCATCTGTGCAAACGGG	A2M
 ```
 
+It's essentially the pyCROQUET-formatted library with the headers stripped to leave a single header row. In the C-SAR configuration file  (`baggen_low_stringency.config`) we use `library_id_column_index = 2` and `library_gene_column_index = 4` to tell C-SAR that the guide identifiers and genes are in column 2 and 4 respectively.
```
Comment (keiranmraine) [inline examples/baggen_2021/README.md:95]: I think like 84 needs to read `The library used with C-SAR is` rather than `The library used with pyCROQUET is`

Comment (vaofford) [reply examples/baggen_2021/README.md:95]: Fixed in 5d7ff6d

**Review by keiranmraine — APPROVED** [review]

**Pull request merged**